calculation/gmhazard_calc/gmhazard_calc/uhs/uhs.py

```diff
@@ -176,15 +176,14 @@ def __get_pSA_values(
             except ValueError:
                 pSA_percentiles.append([np.nan, np.nan])
 
-        try:
-            pSA_branch_values.append(
-                [
-                    cur_branch.exceedance_to_im(cur_exceedance)
-                    for cur_branch in hazard_branches.values()
-                ]
-            )
-        except exceptions.ExceedanceOutOfRangeError as ex:
-            pSA_branch_values.append(np.full(len(hazard_branches), np.nan))
+        temp = []
```

Comment:
Agree with Jason, `temp` is not a good name, maybe `cur_pSA_branch_values` instead?